build.sbt

```diff
@@ -12,8 +12,15 @@ lazy val microservice = Project("preferences-changed-notifier", file("."))
     libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test,
     // https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html
     // suppress warnings in generated routes files
-    scalacOptions += "-Wconf:src=routes/.*:s",
-    scalacOptions += "-feature"
+    scalacOptions ++= List(
+      "-feature",
+      // Silence unused imports in template files
+      "-Wconf:msg=unused import&src=.*:s",
```

Comment:
I think this one is needed for template/html files only

Reply:
I wondered that but I was getting a lot of unused import warnings.

Reply:
ok, you have already added another one for unused imports in routes files 
// Silence unused warnings on Play `routes` files
"-Wconf:src=routes/.*:s"